### DISCUSSION TITLE
我在安装cs50的语库时遇到了一些问题 同时对于wls Ubuntu搭配vscode的操作感到疑惑

### DISCUSSION THREAD

**Discussion by entropy1121** [Xilesun/CS50x#13] [Q&A]
### 你的运行环境是?

cs50.dev

### 请详细描述你的问题，操作步骤。并附上完整的报错信息截图，或其他有效信息。

![image](https://github.com/user-attachments/assets/d2ac21ff-fbff-4941-81ac-4ee0b348633e)
![image](https://github.com/user-attachments/assets/094ea951-38a0-418d-b2ed-cc707338a36b)
![image](https://github.com/user-attachments/assets/9b0d9803-7508-4a7b-bea6-29ec3e426e5b)

我已经尝试完善了环境  在Ubuntu上面设置了如buildessential相关的命令 尝试添加libcs50 后来猜测没成功的原因是没有下载安装包 所以刚刚在解压过程也屡受挫折 目前没有看到Ubuntu上有任何帮助解压的迹象 同时对于今天初次探索和搭建设备感觉进度缓慢 希望老师能够提供帮助 第一次使用github 很多措辞不严谨 见谅

**Comment by 2013xile** [comment]
参考这个文档安装 https://github.com/cs50/libcs50
如果实在有困难，建议先用cs50.dev线上环境。

**Reply by entropy1121** [reply]
> 参考这个文档安装 https://github.com/cs50/libcs50
> 如果实在有困难，建议先用cs50.dev线上环境。

谢谢在本地安装云端插件了 
还有两个问题：
1.check功能的实现必须是按照官方给定的文件设置路径吗？ 我自己想check其他代码貌似做不到这些效果
2.Windows配置wsl Ubuntu对这个学习貌似没有用呢 只记得当时输入过python的下载？ 也没有找到很直接的关系 
如果后续写代码 还是需要Ubuntu远程代理？
其实没有明白云端和远程 貌似他们都不直接关联我的本地

**Reply by 2013xile** [reply]
1. check 需要是官方的作业文件
2. WSL 应该是能关联本地的。使用 WSL 是因为在 Linux 下配置各种环境比直接用 Windows 简单，省去很多配置的麻烦，也能找到更多的资料。
